app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```

Comment:
Remember to add validations to any required field! Accordingly to the card all the fields are required for a book